azure-devops/app-projects/07_tlscert-prod-api-internal-prod-userregistry-pagopa-it.tf

```diff
@@ -0,0 +1,88 @@
+variable "tlscert-prod-api-internal-prod-userregistry-pagopa-it" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "le-azure-acme-tiny"
+      branch_name    = "master"
+      pipelines_path = "."
+    }
+    pipeline = {
+      enable_tls_cert = true
+      path            = "TLS-Certificates\\PROD"
+      dns_record_name = "api.internal"
+      dns_zone_name   = "userregistry.pagopa.it"
+      # common variables to all pipelines
+      variables = {
+        CERT_NAME_EXPIRE_SECONDS = "2592000" #30 days
+      }
+      # common secret variables to all pipelines
+      variables_secret = {
+      }
+    }
+  }
+}
+
+locals {
+  tlscert-prod-api-internal-prod-userregistry-pagopa-it = {
+    tenant_id                           = module.secrets.values["TENANTID"].value
+    subscription_name                   = var.prod_subscription_name
+    subscription_id                     = module.secrets.values["PROD-SUBSCRIPTION-ID"].value
+    dns_zone_resource_group             = local.prod_vnet_rg
+    credential_subcription              = var.prod_subscription_name
+    credential_key_vault_name           = local.prod_key_vault_name
+    credential_key_vault_resource_group = local.prod_key_vault_resource_group
+    service_connection_ids_authorization = [
+      module.PROD-TLS-CERT-SERVICE-CONN.service_endpoint_id,
+    ]
+  }
+  tlscert-prod-api-internal-prod-userregistry-pagopa-it-variables = {
+    KEY_VAULT_SERVICE_CONNECTION = module.PROD-TLS-CERT-SERVICE-CONN.service_endpoint_name,
+    KEY_VAULT_NAME               = local.prod_key_vault_name
+  }
+  tlscert-prod-api-internal-prod-userregistry-pagopa-it-variables_secret = {
+  }
+}
+
+#--------------------------------------------------------------------------------------------------
+
+# change only providers
+#tfsec:ignore:GEN003
+module "tlscert-prod-api-internal-prod-userregistry-pagopa-it-cert_az" {
```

Comment:
check @uolter update that set scheduled triggers in pipelines (no manual change in azure devops)